spec/css/functions/var.hrx

```diff
@@ -0,0 +1,338 @@
+<===> options.yml
+---
+:ignore_for:
+- libsass
+
+<===>
+================================================================================
+<===> css_function/zero_argument/empty/input.scss
+a {b: var()}
+
+<===> css_function/zero_argument/empty/output.css
+a {
+  b: var();
+}
+
+<===>
+================================================================================
+<===> css_function/single_argument/expression/input.scss
+a {b: var(--c)}
+
+<===> css_function/single_argument/expression/output.css
+a {
+  b: var(--c);
+}
+
+<===>
+================================================================================
+<===> css_function/single_argument/rest/input.scss
+$name: --c;
+a {b: var($name...)}
+
+<===> css_function/single_argument/rest/output.css
+a {
+  b: var(--c);
+}
+
+<===>
+================================================================================
+<===> css_function/two_argument/expressions/input.scss
+a {b: var(--c, d)}
+
+<===> css_function/two_argument/expressions/output.css
+a {
+  b: var(--c, d);
+}
+
+<===>
+================================================================================
+<===> css_function/two_argument/dynamic/input.scss
+a {b: var(--c, 1 + 2)}
+
+<===> css_function/two_argument/dynamic/output.css
+a {
+  b: var(--c, 3);
+}
+
+<===>
+================================================================================
+<===> css_function/two_argument/rest/input.scss
+$args: --c, d;
+a {b: var($args...)}
```

Comment:
If I had
```scss
$args: --c, d;
a {b: var($args..., )}
```
or
```scss
$args: --c;
a {b: var($args...,)}
```

Should those cases pass with `a { b: var(--c, d) }` and `a { b: var(--c, ) }` respectively?
or would both be considered errors?

Reply:
That's actually tested on line 275. They're both valid, but they ignore the comma, because Sass generally allows trailing commas in argument lists, maps, and so on.

Reply:
I see, I did notice that test but assumed that `@function var` would make the special handling to work differently, but if both are meant to be similar then this is fine